sdk/src/action-support.js

```diff
@@ -86,7 +86,7 @@ class ActionHandler {
         return call.cancelled;
       },
       get metadata() {
-        return metadata;
+        return metadataObject;
```

Comment:
On line 79-81 we create this object for this purpose, and should return it here.